src/SFA.DAS.AdminService.Web/Controllers/Roatp/Apply/RoatpOrganisationCriminalComplianceChecksController.cs

```diff
@@ -0,0 +1,69 @@
+﻿using MediatR;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Logging;
+using Newtonsoft.Json;
+using SFA.DAS.AdminService.Web.Domain;
+using SFA.DAS.AdminService.Web.Handlers.Gateway;
+using SFA.DAS.AdminService.Web.Infrastructure;
+using SFA.DAS.AdminService.Web.Services.Gateway;
+using SFA.DAS.AdminService.Web.Validators.Roatp;
+using SFA.DAS.AdminService.Web.ViewModels.Roatp.Gateway;
+using SFA.DAS.AssessorService.ApplyTypes.Roatp;
+using System;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace SFA.DAS.AdminService.Web.Controllers.Roatp.Apply
+{
+    [Authorize(Roles = Roles.RoatpGatewayTeam + "," + Roles.CertificationTeam)]
+    public class RoatpOrganisationCriminalComplianceChecksController : RoatpGatewayControllerBase
+    {
+        private readonly IRoatpApplicationApiClient _applyApiClient;
+        private readonly IHttpContextAccessor _contextAccessor;
+        private readonly IRoatpGatewayPageViewModelValidator _gatewayValidator;
+        private readonly IGatewayCriminalComplianceChecksOrchestrator _orchestrator;
+        private readonly ILogger<RoatpOrganisationCriminalComplianceChecksController> _logger;
+
+        public RoatpOrganisationCriminalComplianceChecksController(IRoatpApplicationApiClient applyApiClient, IHttpContextAccessor contextAccessor,
+                                                              IRoatpGatewayPageViewModelValidator gatewayValidator,
+                                                              IGatewayCriminalComplianceChecksOrchestrator orchestrator,
+                                                              ILogger<RoatpOrganisationCriminalComplianceChecksController> logger) : base()
+        {
+            _applyApiClient = applyApiClient;
+            _contextAccessor = contextAccessor;
+            _gatewayValidator = gatewayValidator;
+            _orchestrator = orchestrator;
+            _logger = logger;
+        }
+
+
+        [HttpGet("/Roatp/Gateway/{applicationId}/Page/5-10")]
```

Comment:
this 5-10 code will be going, you'll need to wait for the overview changes to go in.  I expect it to be replaces with 'composition-with-creditors' or similar

Reply:
Replaced with url tags